### PR TITLE
[FIX] Multiple UI adjustment 0.3

### PR DESCRIPTION
**Pull request by hujikol**
# Description

This PR includes the following UIX adjustment:
1. Move grub shop time left to the top on mobile (Discord bug [Link](https://discord.com/channels/880987707214544966/1051458427441922078/1051476055216623626))
2. Fix the Resale modal to be closeable
3. Change chest reward loading state
4. Remove the fountain hovering border
5. Fix claim skill font size
6. Adjust Auctioneer coming soon margin
7. Stadarize underline in syncing modal
8. Exotic seeds coming soon alignment
9. Withdraw the token and item read more adjustment
10. Align crop timer
11. Align tree timer
12. Locked seeds alignment (Discord bug [Link](https://discord.com/channels/880987707214544966/1052671849651261491))
13. Locked building detail adjustment

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

## UI inspection

1.
image

2.
image

3.
image

5.
image
image

7.
image

8. 
image

10.
image

11.
image

12. 
image

13. 
image

# Checklist:

- [x] Title of the PR is relevant and is prefixed with [FEAT], [CHORE], or [FIX]
- [x] Screenshot if it includes any UI changes
- [x] I have read the contributing guidelines and agree to the T&Cs
- [x] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by stevew00dy** [comment]
Thank you so much for these QOL updates, they are really apprecaited.

**Comment by hujikol** [comment]
tag me please if this PR ready to be reviewed so I can merged the changes and not always merging

**Comment by c-py** [comment]
@hujikol Taking a look at this PR now 😁

File: src/features/game/components/Syncing.tsx [hunk 1]
```diff
@@ -14,8 +14,8 @@ export const Syncing: React.FC = () => {
       <span className="text-shadow text-xs text-center">
         Please bear with us while we sync all of your data on chain.
       </span>
-      <span className="text-shadow text-xs text-center underline mt-2">
-        Do not refresh your browser
```
Comment (c-py) [inline src/features/game/components/Syncing.tsx:18]: Would it be possible to remove the capital letters from this modal?

`Do not refresh your browser`

File: src/features/goblins/bank/components/WithdrawItems.tsx [hunk 2]
```diff
@@ -249,7 +249,16 @@ export const WithdrawItems: React.FC<Props> = ({
         </div>
 
         <span className="text-sm mb-4">
-          Once withdrawn, you will be able to view your items on OpenSea.
+          Once withdrawn, you will be able to view your items on OpenSea,{" "}
```
Comment (c-py) [inline src/features/goblins/bank/components/WithdrawItems.tsx:252]:
```suggestion
          Once withdrawn, you will be able to view your items on OpenSea.{" "}
```

I think the capital letter on `Read more` looks a bit odd in the middle of the sentence 😅 

I'm not sure which one is the most suitable:

`Once withdrawn, you will be able to view your items on OpenSea, read more.`

`Once withdrawn, you will be able to view your items on OpenSea; Read more.`

`Once withdrawn, you will be able to view your items on OpenSea. Read more.`

**Review by c-py — CHANGES_REQUESTED** [review]
Great additions @hujikol! They are working well 🚀 .

I have left two pieces of small feedback 😁 

I think for next time, splitting the PR up into individual items will make it easier to review. As if there is any feedback for a single change, it wont block all the other changes from being merged.

File: src/features/goblins/bank/components/WithdrawItems.tsx (hunk 2 above)
Comment (hujikol) [reply src/features/goblins/bank/components/WithdrawItems.tsx:252]: i think i will go for `Once withdrawn, you will be able to view your items on OpenSea. Read more.`

**Comment by hujikol** [comment]
> Great additions @hujikol! They are working well 🚀 .
> 
> I have left two pieces of small feedback 😁
> 
> I think for next time, splitting the PR up into individual items will make it easier to review. As if there is any feedback for a single change, it wont block all the other changes from being merged.

I thought it would spam the PR tab.. so I'm sorry 😭 I will split things up next time

**Review by c-py — APPROVED** [review]
Amazing! Thank you for the great contributions @hujikol ❤️

**Pull request merged**